web/shapeworks/src/store/methods.ts

```diff
@@ -153,6 +159,37 @@ export const loadReconstructedSamplesForProject = async (type: string, id: numbe
     }
 }
 
+export const loadDeepSSMDataForProject = async (id: number) => {
+    if (selectedProject.value) {
+        const results = await Promise.all([
+            await getDeepSSMResultForProject(
+                selectedProject.value.id
+            ),
+            await getDeepSSMAugPairsForProject(
+                selectedProject.value.id
+            ),
+            await getDeepSSMTrainingPairsForProject(
+                selectedProject.value.id
+            ),
+            await getDeepSSMTrainingImagesForProject(
+                selectedProject.value.id
+            ),
+            await getDeepSSMTestImagesForProject(
+                selectedProject.value.id
+            )]
+        )
+
+        console.log(results[0], results[1], results[2], results[3])
```

Comment:
Am I correct to assume that in the follow-up PR, we will replace this log statement with setting some store var?